examples/asset/remote_asset_loader.rs

```diff
@@ -0,0 +1,167 @@
+//! Implements a remote asset loader.
+//! An [`AssetReader`] is what the asset server uses to read the raw bytes of assets.
+//! This example also showcase how to use [`AssetWriter`], thus will won't work on
+//! wasm or Android, since those platforms doesn't have [`AssetWriter`].
```

Comment:
I think it's worth changing this to say that it won't work because wasm and android don't have a default `AssetWriter`. After all, it is possible to implement a custom `AssetWriter` for wasm or android, we just don't have one.